core/src/main/java/hudson/model/Actionable.java

```diff
@@ -204,24 +209,31 @@ public boolean removeActions(@Nonnull Class<? extends Action> clazz) {
      *
      * @param clazz the type of actions to replace
      * @param a     the action to replace with
+     * @return {@code true} if this actions changed as a result of the call
      * @since FIXME
      */
     @SuppressWarnings({"ConstantConditions", "deprecation"})
     @SuppressFBWarnings("RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE")
-    public void replaceActions(@Nonnull Class<? extends Action> clazz, Action a) {
+    public boolean replaceActions(@Nonnull Class<? extends Action> clazz, Action a) {
```

Comment:
As noted [here](http://stackoverflow.com/a/3589948/12916) this would not be binary-compatible even if the method were `final`, which it is not. You must restore the `void` return type. (Anyway you show no non-test callers using the return value and I doubt there would be any.)